Reworded the CONTRIBUTING.md doc (#16849)
bevyengine/bevy: CONTRIBUTING.md

# Contributing to Bevy

> If you'd like to help build Bevy and are looking for a place to start, you can find an
> [introduction here](https://bevyengine.org/learn/contribute/introduction), that can help point
> you in the right direction.  Give it a read, and thanks for contributing!

Reviewer: ```suggestion
If you'd like to help build Bevy, start by reading this
[introduction](https://bevyengine.org/learn/contribute/introduction). Thanks for contributing!
```
